documentation/source/download.rst

```diff
@@ -1,14 +1,76 @@
 .. include:: links.rst
 
-Download
+Download Source Code
+====================
+
+The CDM source code is open source and can be downloaded from `_CDM GitHub`_. All released versions are tagged and can be downloaded from `_CDM GitHub Releases`_.
+
+Versions
 ========
 
-The CDM distribution contains the model source files along with the generated Java and the required dependencies to work with the model. Examples of how you can use this distribution can be found in the CDM Examples also below.
+The CDM code is tagged using semantic versioning and are available on `GitHub Releases <https://github.com/finos/common-domain-model/releases>`_.
+All CDM Binary distribution are created on every release and are published to `Maven Central <https://central.sonatype.com/artifact/org.finos.cdm/cdm-parent>`_.
```

Comment:
"CDM Binary distributions are created on every release and are all published to"